common/database/database_update_manifest.cpp

```diff
@@ -5155,8 +5155,67 @@ ALTER TABLE `npc_emotes` DROP INDEX `emoteid`;
 ALTER TABLE `tasks`
 ADD COLUMN `enabled` smallint NULL DEFAULT 1 AFTER `faction_amount`
 )"
+	},
+	ManifestEntry{
+		.version = 9250,
+		.description = "2023_11_11_guild_features.sql",
+		.check = "SHOW TABLES LIKE 'guild_permissions'",
+		.condition = "empty",
+		.match = "",
+		.sql = R"(
+CREATE TABLE `guild_permissions` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`perm_id` INT(11) NOT NULL DEFAULT '0',
+	`guild_id` INT(11) NOT NULL DEFAULT '0',
+	`permission` INT(11) NOT NULL DEFAULT '0',
+	PRIMARY KEY (`id`) USING BTREE
+)
+ENGINE=InnoDB
+AUTO_INCREMENT=1;
+
+UPDATE guild_ranks SET title = 'Leader' WHERE `rank` = '1';
```

Comment:
I don't have anything in my `guild_ranks` table locally. How is this populated for me otherwise?

Reply:
My guild_ranks from a peq install has ranks for 0 thru 8 though only 0, 1, 2 have names of Member, Officer, Leader.  This structure should be created from existing guild code.  This sql updates the rankes to be RoF2 standards.  Not sure why your local would be empty, unless there are no guilds?  I may need to update for that consideration?  Please let me know.